corehq/apps/accounting/invoicing.py

```diff
@@ -347,12 +347,13 @@ def __init__(self, date_start, date_end, account, recipients=None):
         self.subscriptions = {}
 
     def create_invoice(self):
-        for subscription in self.account.subscription_set.all():
-            if should_create_invoice(subscription, subscription.subscriber.domain, self.date_start, self.date_end):
-                if subscription.plan_version in self.subscriptions:
-                    self.subscriptions[subscription.plan_version].append(subscription)
+        for sub in self.account.subscription_set.all():
+            if not sub.do_not_invoice \
```

Comment:
Just checking: is ```do_not_invoice``` the correct field to consider?  For example, there could be community subscriptions with ```do_not_invoice == False```.  Do we want to bill for users there?

Reply:
That's a really good question. I assumed that we would still want to bill for community subscriptions on customer enterprise accounts. But if we always want to ignore all community subscriptions, then this isn't the right approach

Reply:
I'm not sure what Julie had in mind, should confirm that we're all on the same page.